tools/trustee/src/cli.rs

```diff
@@ -0,0 +1,169 @@
+
+        config.admin.auth_public_key = Some(public_path);
+    }
+
+    if !config.http_server.insecure_http {
+        if config.http_server.private_key.is_none() {
+            let (private_path, _) = ensure_https_key_pair(trustee_home_dir)?;
+
+            config.http_server.private_key = Some(private_path);
+        }
+
+        if config.http_server.certificate.is_none() {
+            let private_key = config.http_server.private_key.as_ref().unwrap();
+            config.http_server.certificate =
+                Some(ensure_https_cert(trustee_home_dir, private_key)?);
+        }
+    }
+
+    if allow_all {
+        warn!("Using policy allow_all. This is for development only.");
+        config.policy_engine.policy_path = trustee_home_dir.join("allow_all.rego");
+        std::fs::write(
+            &config.policy_engine.policy_path,
+            include_bytes!("../../../kbs/sample_policies/allow_all.rego"),
+        )?;
+    } else if !config.policy_engine.policy_path.exists() {
+        config.policy_engine.policy_path = trustee_home_dir.join("deny_all.rego");
+        std::fs::write(
+            &config.policy_engine.policy_path,
+            include_bytes!("../../../kbs/sample_policies/deny_all.rego"),
+        )?;
+    }
+
+    debug!("Config: {:?}", config);
+    Ok(config)
+}
+
+async fn trustee_run(config: KbsConfig) -> Result<()> {
+    let api_server = ApiServer::new(config).await?;
+    api_server.server()?.await?;
+    Ok(())
+}
+
+#[derive(Debug, Parser)]
+pub(crate) enum Commands {
+    /// Generate a new key pair
+    Keygen {
+        /// Output file for the private key
+        #[arg(short = 'f')]
+        output_file: Option<PathBuf>,
+    },
+    /// Launch Trustee
+    Run {
+        /// Configuration file
+        #[arg(long)]
+        config_file: Option<PathBuf>,
+        /// Use built-in policy to allow all (development only).
+        /// If neither this nor a policy file is provided, the default policy is to deny all.
+        #[arg(long)]
+        allow_all: bool,
+    },
+}
+
+#[derive(Debug, Parser)]
+#[command(author, version, about, long_about = None)]
+struct Cli {
+    #[command(subcommand)]
+    command: Commands,
+    /// Path to the directory for storing config and data
+    #[arg(long, value_name = "PATH", env = "TRUSTEE_HOME")]
+    home: Option<PathBuf>,
+}
+
+pub async fn cli_default() -> Result<()> {
+    let cli = Cli::parse();
+
+    let trustee_home_dir = cli.home.unwrap_or_else(|| {
+        if Uid::effective().is_root() {
+            "/opt/confidential-containers".into()
+        } else {
+            home_dir().unwrap_or_default().join(".trustee")
+        }
+    });
+
+    if !trustee_home_dir.exists() {
+        std::fs::create_dir_all(&trustee_home_dir)?;
+    }
+
+    match cli.command {
+        Commands::Keygen { output_file: out } => {
+            let out = out.unwrap_or_else(|| trustee_home_dir.join("key"));
+            trustee_keygen(&out)?;
+        }
+        Commands::Run {
```

Comment:
May want to make this enum variant more explicit in terms of what it is running.